client/src/js/modules/cell/views/beamlines.js

```diff
@@ -9,6 +9,11 @@ define(['marionette', 'utils',
         template: false,
         modelEvents: { 'change': 'render' },
 
+        ascsort: function(a,b) {
```

Comment:
Since this is a static function, and does not depend on the current context, could this be moved to an utils file?

Otherwise, would `.sort().reverse()` work? It is slightly less efficient, however

Reply:
I decided it was easier to have the database do the sorting. I had some trouble with the plotting library not liking undefined combinations of year and beamline, so added some code to set them all to 0 and then replacing with the real count.

To test:
* Go to /statistics/pdb
* Check the stacked bar charts labelled "Depositions by beamline / year" look ok (up to 2020 when it was last updated)